src/sdk/modules/smartSessionsValidator/decorators/trustAttesters.ts

```diff
@@ -3,7 +3,7 @@ import type { Chain, Client, Hex, Transport } from "viem"
 import { sendUserOperation } from "viem/account-abstraction"
 import { getAction, parseAccount } from "viem/utils"
 import { AccountNotFoundError } from "../../../account/utils/AccountNotFound"
-import { MOCK_ATTESTER_ADDRESS, REGISTRY_ADDRESS } from "../../../constants"
+import { REGISTRY_ADDRESS } from "../../../constants"
```

Comment:
pretty sure this whole file can be removed now. Please remove it and all references to it and see if tests pass. I expect they will